common/test/src/ui/tst_MapDocument.cpp

```diff
@@ -462,6 +463,73 @@ TEST_CASE_METHOD(MapDocumentTest, "canUpdateLinkedGroups")
     document->canUpdateLinkedGroups(kdl::vec_static_cast<mdl::Node*>(entityNodes)));
 }
 
+TEST_CASE_METHOD(MapDocumentTest, "newDocument")
+{
+  const auto fgdContent = R"(
+    @SolidClass = worldspawn : "World entity" [
+      some_default_prop(string) : "" : "value"
+    ]
+  )";
+
+  constexpr auto configTemplate = R"(
+    {{
+        version: 9,
+        name: "testNewDocument",
+        icon: "Icon.png",
+        "fileformats": [ {{ "format": "Standard" }} ],
+        "filesystem": {{
+            "searchpath": ".",
+            "packageformat": {{ "extension": ".pak", "format": "idpak" }}
+        }},
+        "materials": {{
+            "root": "textures",
+            "format": {{ "extensions": [".jpg"], "format": "image" }}
+        }},
+        "entities": {{
+            "definitions": [ "WorldSpawnOnly.fgd" ],
+            "defaultcolor": "0.6 0.6 0.6 1.0",
+            "setDefaultProperties": {}
+        }},
+        "tags": {{ "brush": [], "brushface": [] }},
+        "faceattribs": {{ "surfaceflags": [], "contentflags": [] }}
+    }}
+  )";
+
+  SECTION("Default worldspawn properties")
+  {
+    using T = std::tuple<bool, std::vector<mdl::EntityProperty>>;
+    const auto& [setDefaultProperties, expectedEntityProperties] = GENERATE(values<T>({
+      {false,
+       {{mdl::EntityPropertyKeys::Classname,
+         mdl::EntityPropertyValues::WorldspawnClassname}}},
+      {true,
+       {{mdl::EntityPropertyKeys::Classname,
+         mdl::EntityPropertyValues::WorldspawnClassname},
+        {"some_default_prop", "value"}}},
+    }));
+
+    const auto configContent =
+      fmt::format(configTemplate, setDefaultProperties ? "true" : "false");
+
+    const auto gameName = "testNewDocument";
+    const auto dirPath = std::filesystem::current_path() / "fixture/games" / gameName;
+
+    auto environment = io::TestEnvironment{dirPath.string(), [&](auto& env) {
```

Comment:
This bit is not formatted correctly. Please reformat the file with clang-format 18 (I use 18.1.8).

Reply:
Luckily isn't related to the clang-format version. I'm using version 19.1.2 (the one that comes with vscode-cpptools) but the thing is I have it set to format only modified lines (to avoid modifying external libraries with other formatting guidelines). Because I did a commit previously, it wasn't catch.